docs/docs/ecosystem/roadmap.md

```diff
@@ -3,31 +3,122 @@ id: roadmap
 title: Roadmap
 ---
 
+Ory is in active development.
```

Comment:
```suggestion
Ory is actively being developed. The [maturity of our software](LINK TO THE MATURITY PAGES) varies from product to product.
```

Reply:
Good point!
Should we give that page (https://www.ory.sh/docs/ecosystem/versioning) also an update?

The maturity section seems mostly fine, but what comes below is kind of outdated? Or maybe move that compatibility section?